pkg/motor/query.go

```diff
@@ -120,11 +120,11 @@ func (mtr *motorNodeImpl) QueryWhereIs(req mt.QueryWhereIsRequest) (*mt.QueryWhe
 }
 
 func (mtr *motorNodeImpl) QueryWhereIsByCreator(req mt.QueryWhereIsByCreatorRequest) (*mt.QueryWhereIsByCreatorResponse, error) {
-	resp, err := mtr.GetClient().QueryWhereIsByCreator(req.Creator, req.Pagination)
+	resp, err := mtr.GetClient().QueryWhereIsByCreator(req.Creator, nil)
```

Comment:
its not accurate at this time due to key indexes in the keeper there is an issue for this: https://github.com/sonr-io/sonr/issues/565 i should put back the pagination instance if provided. as it will only effect requests that explicitly define a pagination limit.